1/README.md

```diff
@@ -21,16 +22,36 @@ This Specification is a [free and open standard](http://www.digistan.org/open-st
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
 
 ## Tango Specification
-The TANGO control system is a device-oriented controls toolkit for controlling any kind of hardware or software and building SCADA systems.
-
+The TANGO control system is a toolkit for building distributed control systems which are device oriented. 
 
 ### Goals
 
-Tango aims to 
+Tango aims to be extremely flexible and can be used to interface any kind of hardware or software system which can be accessed via one of the software languages supported by Tango (C++, Python, Java and JavaScript).
+
+The goal of Tango is to provide a way of representing all physical and software systems which need to be controlled by a control system as Tango Devices.
+
+Devices are software components which have Commands (methods), Attributes (data) and Properties (configuration) and are accessible on the network or locally.
+
+A Tango control system consists of a collection of Tango Devices, an (optional) Database and a set of applications for configuring and controlling the Devices via well defined Application Programmers Interfaces (APIs). Tango defines and maintains official APIs for C++, Python, Java, JavaScript and REST.
+
+The Tango Database serves the purpose of defining and declaring all Devices in the control system and their configuration parameters. The Database also stores the network address of each Device and serves as Naming Service for clients who need to establish a connection to any or all Devices.
```

Comment:
I'd remove "stores the network addresses" as it is likely to me misinterpreted here... rephrase as "The Database also serves as Naming Service for clients who need to establish a connection to any or all Devices".